galaxy_ng/app/api/ui/viewsets/user.py

```diff
@@ -9,16 +11,34 @@
 from galaxy_ng.app.api import base as api_base
 
 
+class UserFilter(filterset.FilterSet):
+    sort = filters.OrderingFilter(
+        fields=(
+            ('username', 'username'),
+            ('email', 'email'),
+            ('first_name', 'first_name'),
+            ('date_joined', 'date_joined')
+        )
+    )
+
+    class Meta:
+        model = auth_models.User
+        fields = ('username', 'email', 'first_name', 'date_joined')
+
+
 class UserViewSet(
     mixins.ListModelMixin,
     mixins.RetrieveModelMixin,
     mixins.CreateModelMixin,
     mixins.UpdateModelMixin,
+    mixins.DestroyModelMixin,
     api_base.GenericViewSet,
```

Comment:
DRF provides `ModelViewSet` class, that is implemented exactly as 
```
class ModelViewSet(mixins.CreateModelMixin,
                   mixins.RetrieveModelMixin,
                   mixins.UpdateModelMixin,
                   mixins.DestroyModelMixin,
                   mixins.ListModelMixin,
                   GenericViewSet):
```

You could add the following class to `base.py` and use it here.

```

class ModelViewSet(LocalSettingsMixin, viewsets.ModelViewSet):
    pass
```